apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/AplDbVersion.java

```diff
@@ -688,7 +688,7 @@ protected int update(int nextUpdate) {
             case 252:
                 // SHARDING meta-info inside main database
                 apply("CREATE TABLE IF NOT EXISTS shard (shard_id BIGINT NOT NULL, shard_hash VARBINARY, " +
-                        "shard_height INT not null default 0, shard_state BIGINT default 0, zip_hash_crc VARBINARY, prunable_zip_hash VARBINARY DEFAULT NULL, generator_ids ARRAY DEFAULT NULL)");
+                        "shard_height INT not null default 0, shard_state BIGINT default 0, zip_hash_crc VARBINARY, generator_ids ARRAY DEFAULT NULL, block_timeouts ARRAY DEFAULT NULL, block_timestamps ARRAY DEFAULT NULL, prunable_zip_hash VARBINARY DEFAULT NULL)");
```

Comment:
What is that used for ( block_timeouts ARRAY DEFAULT NULL ) ? Why we need that new column in shard table ?

Reply:
block_timeouts belong to old columns added earlier, required to process first 3 shard blocks

Reply:
For shard compatibility we should arrange field in order of adding to db